sql/core/src/test/resources/sql-tests/inputs/udf/udf-inline-table.sql

```diff
@@ -0,0 +1,54 @@
+-- This test file was converted from intersect-all.sql.
+-- Note that currently registered UDF returns a string. So there are some differences, for instance
+-- in string cast within UDF in Scala and Python.
+
+-- single row, without table and column alias
+select * from values ("one", 1);
+
+-- single row, without column alias
+select * from values ("one", 1) as data;
+
+-- single row
+select udf(a), b from values ("one", 1) as data(a, b);
```

Comment:
I think `values ("one", udf(1))` is not allowed as of SPARK-28291. We can add that test here, and comment them with linking `SPARK-28291` JIRA.

Reply:
See `udf-aggregates_part1.sql` to  check how I commented them.

Reply:
I added a comment at the top of the file.